solr/core/src/test/org/apache/solr/response/JSONWriterTest.java

```diff
@@ -314,4 +315,35 @@ public void testConstantsUnchanged() {
     assertEquals("arrntv", JSONWriter.JSON_NL_ARROFNTV);
     assertEquals("json.wrf", JSONWriter.JSON_WRAPPER_FUNCTION);
   }
+
+  @Test
+  public void testWfrJacksonJsonWriter() throws IOException {
```

Comment:
I love the test...  Would it make sense to break out these tests?   Move to the JacksonJsonResponseWriterTest.java and JsonResponseWriterTest.java?